pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter.go

```diff
@@ -92,7 +93,14 @@ func (admitter *VMsAdmitter) Admit(ar *admissionv1.AdmissionReview) *admissionv1
 		return webhookutils.ToAdmissionResponseError(err)
 	}
 
-	causes := ValidateVirtualMachineSpec(k8sfield.NewPath("spec"), &vm.Spec, admitter.ClusterConfig, accountName)
+	// TODO: Do we want to be able to change flavors on existing VMs?
```

Comment:
I don't think it would be a problem to change the flavor. If updating with a different flavor collides with the VM spec's values, then i'd expect the update to fail. that would happen regardless if the flavor is present at VM creation time or updated later on though. 

 I can't think of a technical reason why changing a flavor on a VM should be avoided.

Reply:
I meant it as a design question. 
The current code allows it. If we did not want to allow it, I would need to add more code.